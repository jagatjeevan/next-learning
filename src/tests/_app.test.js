import renderer from 'react-test-renderer';
import MyApp from '../../pages/_app';

it('renders a snapsÌµhot', () => {
  const tree = renderer.create(<MyApp />).toJSON();
  expect(tree).toMatchSnapshot();
});
