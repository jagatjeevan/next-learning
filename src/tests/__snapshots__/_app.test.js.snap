// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapsÌµhot 1`] = `
<h1>
  App
</h1>
`;
